common/beluga_demo_gazebo/launch/bringup.launch.py

```diff
@@ -16,67 +16,83 @@
 
 from ament_index_python.packages import get_package_share_directory
 from launch import LaunchDescription
+from launch.actions import AppendEnvironmentVariable
 from launch.actions import IncludeLaunchDescription
 from launch.launch_description_sources import PythonLaunchDescriptionSource
 from launch.actions import DeclareLaunchArgument
-from launch_ros.substitutions import FindPackageShare
-from launch.substitutions import LaunchConfiguration, PathJoinSubstitution
+from launch.substitutions import LaunchConfiguration
 
 
 def generate_launch_description():
-    worlds_install_folder = os.path.join(
-        get_package_share_directory("beluga_demo_gazebo"),
-        "worlds",
-    )
-    available_worlds = os.listdir(worlds_install_folder)
+    world_paths = [
+        os.path.join(get_package_share_directory("gz_ekumen_worlds"), "worlds"),
```

Comment:
```suggestion
        os.path.join(
            get_package_share_directory("gz_ekumen_worlds"),
            "worlds",
        ),
```

Reply:
@glpuga this is actually [`black`](https://github.com/Ekumen-OS/beluga-demos/blob/5c0d6830c33a251004ed80fada295702cac2d335/.pre-commit-config.yaml#L41-L45) formatting.

Reply:
If you put a commat after the last element, black will arrange it vertically.